compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/generators/DataClassMembersGenerator.kt

```diff
@@ -67,37 +69,93 @@ class DataClassMembersGenerator(val components: Fir2IrComponents) {
                 valueParameterDescriptor.bind(this)
             }
 
+        private val equalsName = Name.identifier("equals")
+        private val hashCodeName = Name.identifier("hashCode")
+        private val toStringName = Name.identifier("toString")
```

Comment:
Nit (to myself): It might be better to move these into companion object, like I did at https://github.com/JetBrains/kotlin/pull/3300/commits/8ea1b876f445ec11456d1e962ffa93e0e003f8e2#diff-ad4af4ebe1770279317802646a8911e7

Reply:
Done while rebasing: https://github.com/JetBrains/kotlin/pull/3294/commits/1b740726ccc2bea286c7c74e53483abd8ea4e603